pkg/server/start.go

```diff
@@ -65,6 +65,7 @@ func NewCommandStartAdapterServer(stopCh <-chan struct{}) *cobra.Command {
 		MetricsAddress:                    ":7979",
 		ZMONTokenName:                     "zmon",
 		CredentialsDir:                    "/meta/credentials",
+		HostnameRPSMetricName:             "skipper_serve_host_duration_seconds_count",
```

Comment:
This might be a mistake, if I only activate this collector when I pass the metric name, I should not have a default metric name. Something I could do to have a default metric name is to have a second flag that the only purpose is to activate this collector. The other option would be not having a default value for this flag. I'll decide it as the PR progresses.